tasks/compile-stylesheets.mjs

```diff
@@ -62,29 +59,39 @@ export async function compileStylesheet ([modulePath, { srcPath, destPath }]) {
 
   // Render Sass
   if (!isPackage) {
-    ({ css, map } = await sassRender({
-      file: moduleSrcPath,
-      outFile: moduleDestPath,
+    ({ css, sourceMap: map } = await compileAsync(moduleSrcPath, {
+      alertColor: true,
 
       // Turn off dependency warnings
       quietDeps: true,
 
       // Enable source maps
       sourceMap: true,
-      sourceMapContents: true,
+      sourceMapIncludeSources: true,
 
       // Resolve @imports via
-      includePaths: [
+      loadPaths: [
         join(paths.src, 'govuk'),
         join(paths.node_modules, 'govuk_frontend_toolkit/stylesheets'),
         paths.node_modules
-      ]
+      ],
+
+      // Sass custom logger
+      logger: logger({
+        suppressed: [
+          'Using / for division is deprecated and will be removed in Dart Sass 2.0.0.',
```

Comment:
We can list log messages we'd like to suppress here

Since fixing "Using / for division" would require recreating `math.div()` we can instead silence it for now

It can be revisited once we drop LibSass and Node Sass